lib/common/utils.js

```diff
@@ -156,6 +156,16 @@ export const relativeTo = function relativeTo() {
   // Resolve path
   let _path = r(...args)
 
+  // Keep webpack inline loader intact
+  if (_path.indexOf('!') !== -1) {
+    const [loader, __path, ...rest] = path.split('!')
```

Comment:
What if treat last item of splitted path as real path ?

```js
const [loader, ...rest] = path.split('!')
const __path = rest.pop()
```

Reply:
Totally true. I somehow missed that someone could use multiple loaders and assumed that a second `!` would mean s.o. used it in a file/folder name.